mlir/lib/ExecutionEngine/ArmRunnerUtils.cpp

```diff
@@ -0,0 +1,60 @@
+//===- ArmRunnerUtils.cpp - Utilities for configuring architecture properties //
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
Could you add links to some relevant docs? Sounds like `PR_SVE_SET_VL` and `PR_SME_SET_VL` are key for this lib.